code/modules/mob/living/basic/space_fauna/morph.dm

```diff
@@ -58,6 +58,7 @@
 
 	AddElement(/datum/element/ai_retaliate)
 	AddElement(/datum/element/content_barfer)
+	AddElement(/datum/element/prevent_attacking_of_types, GLOB.typecache_general_bad_hostile_attack_targets, "this tastes awful!") // Monkestation edit
```

Comment:
ADDITION is for adding lines, edit is for when you remove something and replace it with something else, or add onto another line
```suggestion
	AddElement(/datum/element/prevent_attacking_of_types, GLOB.typecache_general_bad_hostile_attack_targets, "this tastes awful!") // MONKESTATION ADDITION
```